jenkins/aws/manageDocker.sh

```diff
@@ -272,9 +272,20 @@ case ${DOCKER_OPERATION} in
             DOCKERFILE="${AUTOMATION_BUILD_DEVOPS_DIR}/docker/Dockerfile"
         fi
 
-        # Perform the build
-        docker build -t "${FULL_DOCKER_IMAGE}" -f "${DOCKERFILE}" "${AUTOMATION_BUILD_DIR}"
-        RESULT=$?
+        if [[ -n ${DOCKER_GITHUB_SSH_KEY_FILE} ]]; then
+            if [[ -f "${DOCKER_GITHUB_SSH_KEY_FILE}" ]]; then
+                # Perform the build
+                info "build docker image with SSH_KEY argument"
+                docker build -t "${FULL_DOCKER_IMAGE}" -f "${DOCKERFILE}" "${AUTOMATION_BUILD_DIR}" --build-arg SSH_KEY="$(cat ${DOCKER_GITHUB_SSH_KEY_FILE})"
```

Comment:
Just looking at how the variable is retrieved, the cat of the file as a variable could run into issues if the file is a multiline file. Is it safe to assume that the ssh key file won't have more than a single line?

Reply:
@roleyfoley it works fine at the moment with a multiline file, so looks like it shouldn't be an issue.